store/postgres/src/relational_queries.rs

```diff
@@ -1995,8 +2038,24 @@ pub struct RevertEntityData {
     pub id: String,
 }
 
+impl RevertEntityData {
+    /// Convert primary key ids from Postgres' internal form to the format we
+    /// use by stripping `\\x` off the front of bytes strings
+    fn convert(table: &Table, mut data: Vec<RevertEntityData>) -> Vec<RevertEntityData> {
```

Comment:
The method name could be more descriptive.

Reply:
Changed it to `bytes_as_str` since it's doing the same thing as that helper, but on a collection of stuff.